magento/magento2
# Implemented skipping price indexation process on `catalogrule_product` indexer, when there is no active catalog rule

Implemented skipping price indexation process on `catalogrule_product` indexer, when there is no active catalog rule

### Description (*)


This is a fast fix for specific situation, when `catalogrule_product` index is on schedule mode and there is no active catalog rule to index. That problems enlarges on M2 instanced with large amount of products and dynamic data.



### Manual testing scenarios

### Fixed Issues (if relevant)

1. Fixes magento/magento2#34784

### Manual testing scenarios (*)

1. Add some ids (or update some product attributes) into catalogrule_product_cl table.
2. Run indexer_update_all_views CRON job.
3. Catalog rule index data should be valid.

Retry steps with active catalogrule and when catalog rule is marked from active to inactive and vice versa. 

### Questions or comments


### Contribution checklist (*)
